bootstrap/sql/postgresql/create_tables.sql

```diff
@@ -0,0 +1,344 @@
+-- THE NAMES OF THE TABLE COLUMNS MUST MATCH THE NAMES OF THE CORRESPONDING CLASS MODEL FIELDS;
+
+CREATE TABLE IF NOT EXISTS dashboard (
```

Comment:
In one of the other PRs you have renamed these tables and few classes to a standard. Do we want to do that here too?